### ISSUE TITLE
Soon to be deprecated?

### ISSUE THREAD

**Issue by Aditya94A** [ro-savage/react-scripts-cssmodules#17]
Just wanted to confirm that with the impending arrival of react-scripts v2.0.0 (beta out now), react-scripts-cssmodules will no longer be needed, right? CRA now has built in support for CSS modules 🎉 

https://github.com/facebook/create-react-app/issues/5103

**Comment by ro-savage** [comment]
That is correct. Once CRA v2 is released, this will only be updated for major bug fixes. And the readme will be updated to suggest everyone use CRA v2.
